File: Back/src/Proeventos/Models/Evento.cs
namespace Proeventos.Models;

public class Evento 
{
    public int EventoId { get; set; }
    public string Local { get; set; }
    public string Data { get; set; }
    public string Theme { get; set; }
    public int NumberOfPeoples { get; set; }
    public string Lote { get; set; }
    public string imgURL { get; set; }
}

File: Front/Proeventos-app/src/app/models/Evento.ts
import {Batch} from "./Batch";
import {RedeSocial} from "./RedesSociais";
import {EventSpeaker} from "./EventSpeaker";
import {Palestrante} from "./Palestrante";

export interface Evento {
  date?: Date;
  email: string;
  id: number;
  base64: string;
  local: string;
  numberOfPeoples: number;
  phone: string;
  socialMedias: RedeSocial[];
  speakerId: number;
  batches: Batch[];
  theme: string;
}

